client/src/commands/submitTransaction.ts

```diff
@@ -78,13 +78,13 @@ export async function submitTransaction(evaluate: boolean, treeItem?: Instantiat
     }
 
     let args: Array<string> = [];
-    const argsString: string = await UserInputUtil.showInputBox('optional: What are the arguments to the transaction, (comma seperated)');
+    const argsString: string = await UserInputUtil.showInputBox('optional: What are the arguments to the transaction, (e.g. \'["arg1", "arg2"]\')', '\'[]\'');
```

Comment:
the e.g. shouldn't have the single quotes, and also the placeholder doesn't have the single quotes